infrastructure/firecracker/types.go

```diff
@@ -167,3 +176,12 @@ type Metadata struct {
 
 // InstanceState is a type that represents the running state of a Firecracker instance.
 type InstanceState string
+
+const (
+	// InstanceStateNotStarted the instance hasn't started running yet.
+	InstanceStateNotStarted InstanceState = "Not started"
+	// InstanceStateRunning the instance is running.
+	InstanceStateRunning InstanceState = "Running"
+	// InstanceStatePaused the instance is currently paused.
+	InstanceStatePaused InstanceState = "Paused"
+)
```

Comment:
lol how many ways of noting status do we have now? i think i count 3

Reply:
I know :rofl:   I just moved this down to be with their type declaration. We should pickup that bug again thats around the status.